src/assets/images/planets.rs

```diff
@@ -29,11 +29,11 @@ pub enum PlanetSprite {
     GasGiantWorld3,
     GasGiantWorld4,
     GasGiantWorld5,
-    GasGiantWorld6,
-    GasGiantWorld7,
-    GasGiantWorld8,
-    GasGiantWorld9,
-    GasGiantWorld10,
+    // GasGiantWorld6,
```

Comment:
These are different tile sizes to the rest meaning we'd need an if condition for these specific ones... Not ideal...